### PR TITLE
remove custom registry in CI

### PR DESCRIPTION
**Pull request by SimeonEhrig**
test PR to test the integration tests: https://github.com/QEDjl-project/QEDbase.jl/pull/123

### CODE REVIEW

File: .github/workflows/BuildDeployDoc.yml [hunk 1]
```diff
@@ -21,8 +21,6 @@ jobs:
           version: "1.10"
       - name: Add custom registry 
         run: |
```
Comment (AntonReinhard) [inline .github/workflows/BuildDeployDoc.yml:23]: Lines 22 and 23 should be deleted too

```suggestion
```

**Review by AntonReinhard — APPROVED** [review]
Just the one suggestion, otherwise looking good

**Comment by SimeonEhrig** [comment]
> Just the one suggestion, otherwise looking good

done

**Review by AntonReinhard — APPROVED** [review]
:+1:

**Pull request merged**